toolchains/solidity/core/crates/linter-server/Cargo.toml

```diff
@@ -11,5 +11,5 @@ crate-type = ["cdylib", "rlib"]
 
 [dependencies]
 osmium-libs-lsp-handler = { path = "../../../../../libs/lsp-handler", version = "0.0.1" }
-solidhunter-lib = { path = "../linter-lib", version = "0.0.2" }
+solidhunter-lib = { path = "../linter-lib", version = "0.0.3" }
```

Comment:
Should be incremented as minor since it is in pre release and this is a breaking change

Reply:
ok, but there is no breaking change in the linter-lib tho, only added a pub function, the rest is only code cleaning and simplification doesn't change the usage.
Should I still mark it as a minor update ?

Reply:
Ok for the lib, the cli should have the minor increase though (-> 0.1.0)

Reply:
that's already the case then 👍